zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepo.java

```diff
@@ -33,7 +35,7 @@
  */
 public interface NotebookRepo extends Closeable {
 
-  void init(ZeppelinConfiguration zConf) throws IOException;
+  void init(ZeppelinConfiguration zConf, Gson gson) throws IOException;
```

Comment:
I think this is a public API, and users may have custom implementations, such change will break them. Additionally, we'd better avoid leaking the third-party lib classes as much as possible.